### ISSUE TITLE
infinite recursion opening RedirectNode in admin with django-polymorphic >= 0.8

### ISSUE THREAD

**Issue by benkonrath** [django-fluent/django-fluent-pages#110]
Opening a fluent_pages RedirectNode in the admin doesn't work with django-polymorphic >= 0.8. Here's the error:

```
RuntimeError at /admin/fluent_pages/page/10/
maximum recursion depth exceeded in cmp
```

I see this problem with django-polymorphic 0.9.1 and 0.8.1. The problem is not present with django-polymorphic 0.7.2.

I'm using these fluent pages package versions:

```
django==1.8.12
django-fluent-pages==0.9
django-fluent-utils==1.2.3
django-parler==1.6.2
django-mptt==0.8.3
django-polymorphic-tree==1.2.3
django-wysiwyg==0.7.1
django-tag-parser==2.1
django-any-urlfield==2.2.1
```

**Comment by vdboor** [comment]
Confirmed! This is likely caused by some change in Django's fieldset handling. I'll check it out

**Comment by benkonrath** [comment]
Thanks. I think I saw this problem on Django 1.6 as well. Not sure if this helps though.

**Comment by benkonrath** [comment]
@vdboor Do you recommend sticking with 0.9 or using 1.0b2? I'm not sure if the problem is fixed by 1.0b2 though.

**Comment by jmurty** [comment]
We are also seeing this issue with _django-polymorphic==0.9.2_.

I may be missing something, but in my quick tests it seems that simply replacing the `RedirectNodeAdmin.fieldsets` attribute with an attribute named `base_fieldsets` instead fixes the infinite recursion. This is per the class documentation for `PolymorphicChildModelAdmin`.

For now we have worked around the issue in our project by monkey-patching `RedirectNodeAdmin` like so, roughly::
```python
if getattr(RedirectNodeAdmin, 'fieldsets', None):
    RedirectNodeAdmin.base_fieldsets = RedirectNodeAdmin.fieldsets
    RedirectNodeAdmin.fieldsets = None
```
